docs/content/gameobject/reading-and-writing-component-data.md

```diff
@@ -73,6 +73,9 @@ public class ReadHealthBehaviour : MonoBehaviour
 2. Send a component update to specify the new component values that your component should be updated to using `Writer.Send(TComponentUpdate update)`.
 </br>(`ISpatialComponentUpdate` types are generated under `<namespace of schema component>.<component name>.Update`.) 
 
+**Known Issue Warning:** 
+- At the moment, updating a component using `Writer.Send` will lead to the SpatialOS GDK updating all properties of the component even if you indicated to only partially update a component. The values of all other properties will be updated to the same value that they already assume. This is a bug and not intended behaviour and will be fixed in an upcoming update.
```

Comment:
* Think we don't need "at the moment" because it's a known issue, so we know it's current.
* Also, avoid prophecies "will" when something happens - just state that it happens.
* Also avoid passive voice where possible (Ie "will be updated")  - better to say what updates it so it's clear whether the user has do do something.
* Suggest we reference updating properties of the component throughout to make it clear (rather than updating a component).
* Avoid the word "bug" (even though it is a bug) as "bug" means bad.
* "Will" is fine where talking about a future change.
* US spelling.

Question: why is this bad - is it just because of inefficiency or something else?

So, I suggest something like...

"When you use `Writer.Send` to update a component's properties, it causes the GDK to update _all_ the properties of the component, even if you have set it to only update _some_ of the properties. Note that GDK updates unchanged properties to the value they already have. 
This is unintended behavior which we will fix in an upcoming update."

Reply:
I like the suggestion. Thanks. :)